docs/book/src/reference/project-config.md

```diff
@@ -2,7 +2,74 @@
 
 ## Overview
 
-The Project Config represents the configuration of a Kubebuilder project. All projects that are scaffolded with the CLI will generate the `PROJECT` file in the projects' root directory.
+The Project Config represents the configuration of a KubeBuilder project. All projects that are scaffolded with the CLI will generate the `PROJECT` file in the projects' root directory. Therefore, it will store all plugins and input data used to generate the project and APIs to better enable plugins to make useful decisions when scaffolding.
+
+## Example
+
+Following an example of an PROJECT file config file which is the result of an project generated with two APIs using the [Deploy Image Plugin][deploy-image-plugin].
```

Comment:
At the bottom of the page we need to add the alias for this link it is here: https://book.kubebuilder.io/plugins/deploy-image-plugin-v1-alpha.html

So we need the relative path which will be something like 
[deploy-image-plugin]: docs/plugins/deploy-image-plugin-v1-alpha.md 

Please, check the other links in the docs to know how to do that

Reply:
Added this